src/as_execution/context.rs

```diff
@@ -14,12 +10,12 @@ use wasmer::{
 use wasmer_middlewares::metering::{self, MeteringPoints};
 use wasmer_types::TrapCode;
 
-pub(crate) struct ASContextModule {
+pub(crate) struct ASContext {
```

Comment:
Add some quick doc about what is a Context?

Reply:
Done